streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockHandlerSuite.scala

```diff
@@ -174,6 +187,172 @@ class ReceivedBlockHandlerSuite
     }
   }
 
+  test("BlockManagerBasedBlockHandler-MEMORY_ONLY-ByteBufferBlock - count messages") {
+    storageLevel = StorageLevel.MEMORY_ONLY
+    // Create a non-trivial (not all zeros) byte array
+    var counter = 0.toByte
+    def incr: Byte = {counter = (counter + 1).toByte; counter;}
+    val bytes = Array.fill[Byte](100)(incr)
```

Comment:
These 3 lines can be replaced by `val bytes = Array.tabulate(100)(i => i.toByte)`